### PR TITLE
#161595888 Rename travel documents

### PR DESCRIPTION
**Pull request by rovilay**
#### What does this PR do?
This PR renames travel documents

#### Description of Task to be completed?
- Convert mockup to react components
- Style react components to match mockup
- Create action, reducer, and saga for document rename
- Write tests for implementation

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- Install dependencies and start the server - `make start`
-  [Start](https://github.com/andela/travel_tool_back/pull/165)  the backend.
- Navigate to [travela](http://travela-local.andela.com:3000) in your browser and login into the app. 
- Click on the  `Document` tab
- Add a document if you don't have any existing document
- On a document row, click on the `ellipses`
- On the menu dropdown, select `Rename`
- A modal will pop up, fill the form and submit

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[#161595888](https://www.pivotaltracker.com/story/show/161595888)

#### Screenshots (if appropriate)
screenshot 2018-11-08 at 4 41 26 pm

#### Questions:
N/A

### CODE REVIEW

File: src/components/DocumentsHeader/DocumentsHeader.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.document-panel-header {
+  box-sizing: border-box;
+  background-color: #f8f8f8;
+  height: 100px;
+  width: 96.3%;
+  @media (max-width: 479px) {
+    width: 94%;
+  }
+}
```
Comment (clintfidel) [inline src/components/DocumentsHeader/DocumentsHeader.scss]: please add a new line

File: src/images/icons/documents-blue.svg [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                <g id="icon/action/settings_24px-copy" transform="translate(0.000000, 2.000000)">
+                    <mask id="mask-2" fill="white">
+                        <use xlink:href="#path-1"></use>
+                    </mask>
+                    <use id="icon/file/folder_24px" fill="#3359DB" fill-rule="nonzero" xlink:href="#path-1"></use>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (clintfidel) [inline src/images/icons/documents-blue.svg]: please add a new line and also replicate this at different places where needed

File: src/views/Documents/__tests__/index.test.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+  });
+
+  it('should should render documents editmodal', (done) => {
+    const currentProps = { ...props, modalType: 'rename document' };
+    const wrapper = mount(<Documents {...currentProps} />);
+    const toggleIcons = wrapper.find('#toggleIcon');
+    const toggleMenuSpy = jest.spyOn(wrapper.instance(), 'toggleMenu');
+    expect(toggleIcons.length).toBe(2);
+
+    // toggleIcons.at(1).simulate('click');
```
Comment (clintfidel) [inline src/views/Documents/__tests__/index.test.js]: please take out commented code

File: src/components/DocumentsHeader/DocumentsHeader.scss (hunk 1 above)
Comment (rovilay) [reply src/components/DocumentsHeader/DocumentsHeader.scss]: Done !

File: src/views/Documents/__tests__/index.test.js (hunk 3 above)
Comment (rovilay) [reply src/views/Documents/__tests__/index.test.js]: Done!

**Comment by Spectrumsun** [comment]
I get an error after renaming 
screenshot 2018-11-08 at 6 50 04 pm 2

**Comment by Spectrumsun** [comment]
can you declare some of the props validation as constant's to reduce code climate error

**Comment by rovilay** [comment]
@spectrum...for the error you're receiving please ensure you're using the [backend] (https://github.com/andela/travel_tool_back/pull/165) of this PR.

**Comment by Spectrumsun** [comment]
Please fix conflicts @rovilay

**Comment by rovilay** [comment]
@Spectrumsun  done!

**Comment by Spectrumsun** [comment]
Good job @rovilay

**Pull request merged**